src/quickgui/plugin/qgsquickfeatureform.qml

```diff
@@ -274,16 +275,20 @@ Item {
           }
         }
 
-        Connections {
-          target: form
-          onAboutToSave: {
-            try {
-              attributeEditorLoader.item.pushChanges()
-            }
-            catch ( err )
-            {}
-          }
-        }
+        // TODO delete or fix - saving works without it
+//        Connections {
+//          target: form
+//          onAboutToSave: {
+//            try {
+//                console.log("onAboutToSave: " + attributeEditorLoader.item)
+//              attributeEditorLoader.item.pushChanges()
+//            }
+//            catch ( err )
+//            {
+//            console.log("ERROR: " + err)
+//            }
+//          }
+//        }
```

Comment:
Is there a reason this needs to be commented out? I think some edit widgets may use the aboutToSave signal to do some work...